### ISSUE TITLE
JS error on Internet Explorer 9

### ISSUE THREAD

**Issue by bastienlm** [algolia/algoliasearch-magento#689]
Hello,

There is a JS error which blocks the selection of the options on a configurable product page.
Example: https://magento.algolia.com/french-cuff-cotton-twill-oxford-570.html

You can't select a size on this product because of the error (in french) 
"`L’objet ne gère pas la propriété ou la méthode « observe »`" raised by prototype.js on line 828.

If we disable Algolia, everything works fine.

Regards,

**Comment by JanPetr** [comment]
Thank you for the report @bastienlm,
we will investigate and come back to you.

**Comment by yhoiseth** [comment]
Hi @JanPetr,

We're experiencing this issue on a recently-installed algoliasearch-magento. See for example http://www.skalanetshop.no/overall-med-hette-bla.html.

However, I noticed that it no longer occures on https://magento.algolia.com/french-cuff-cotton-twill-oxford-570.html.

Any ideas?

**Comment by yhoiseth** [comment]
It seems like we have at least a temporary fix for this issue.

We tried a few approaches:

1. Put the Algolia JavaScript files after our custom ones in `<head>` in order to test whether they were messing up anything. That didn't make any difference.
2. Replaced the contents of `prototype.js` with [version 1.7.3](https://cdnjs.cloudflare.com/ajax/libs/prototype/1.7.3/prototype.min.js) because I read somewhere that Prototype.js 1.7.3 included lots of IE-related fixes. This caused some other errors.
3. Finally, I commented out [the polyfill in `algoliasearch.xml`](https://github.com/algolia/algoliasearch-magento/blob/master/app/design/frontend/base/default/layout/algoliasearch.xml#L16), and that resolved the issue.

I haven't noticed any side effects, but I'm probably missing something.
